tests/e2e/cucumber/features/smoke/applicationMenu.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Application menu
```

Comment:
We decided to skip unit tests and added e2e as this is more reliable and easy to test